wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -285,8 +285,8 @@
 
     public static int USERNAME_MIN_LENGTH = 3;
     public static int USERNAME_MAX_LENGTH = 23;
-    public static int DISPLAY_NAME_MAX_LENGTH = 20;
-    public static int ABOUT_ME_MAX_LENGTH = 120;
+    public static int DISPLAY_NAME_MAX_LENGTH = 25;
+    public static int ABOUT_ME_MAX_LENGTH = 250;
```

Comment:
The current contract for dashpay only allows 20 and 140 characters.  Hopefully this is changed for the next network upgrade, but for now we need to use the smaller numbers otherwise the create/update profile items will fail.

Reply:
Fixed